### PR TITLE
Add Update Selection Menu

### LINKED ISSUE #88: Update cheat or patch selection

### PR DESCRIPTION
**Pull request by illusionyy**
Fixes #88

### CODE REVIEW

**Comment by illusionyy** [comment]
Fixed button prompts

**Comment by illusionyy** [comment]
Commit a584cee832bd4b1b853e42c4bfc2429703b0dd15 broke button prompt and subtitle.
image
image

**Comment by illusionyy** [comment]
Subtitle is created from `Draw_UserCheatsMenu()` + `Draw_UserCheatsMenu_Ani()`.

**Comment by bucanero** [comment]
I cleaned up the code, since most stuff was already there. I can't test right now as I don't have my hardware around, but hopefully the changes still work as expected.

File: source/main.c [hunk 1]
```diff
@@ -70,6 +70,7 @@ const char * menu_pad_help[TOTAL_MENU_IDS] = { NULL,												//Main
 								"\x13 Back",														//View Cheat
 								"\x10 Select    \x13 Back",											//Cheat Option
 								"\x13 Back",														//View Details
+								"\x10 Select    \x13 Back",											//Update Selection
```
Comment (bucanero) [inline source/main.c]: there's no need for this, as there was no need for additional menus . The prompt from `Select cheats` will be used.

**Comment by bucanero** [comment]
> Subtitle is created from `Draw_UserCheatsMenu()` + `Draw_UserCheatsMenu_Ani()`.

if you don't want the `2 Games` subtitle, then a quick fix can be done:

```c
    snprintf(subtitle, sizeof(subtitle), "%zu Games", list_count(list->list));
    if (list->icon_id == header_ico_xmb_png_index) subtitle[0] = 0;
```

**Comment by illusionyy** [comment]
that fixed it, any idea for the button prompt?
image

**Comment by illusionyy** [comment]
Should be just `X Select O Back` like the previous menu. it is using `SetMenu(MENU_PATCHES)` case in `doSaveMenu`.

**Comment by bucanero** [comment]
> Should be just `X Select O Back` like the previous menu. it is using `SetMenu(MENU_PATCHES)` case in `doSaveMenu`.

Maybe a quick hack could be done directly when drawing the help prompt... a bit ugly, but might work. I'll commit a change so you can test if the hack works as expected.

**Comment by illusionyy** [comment]
Workaround works

**Review by bucanero — APPROVED** [review]

**Comment by bucanero** [comment]
> Workaround works

Alright, everything is fixed then. I'll merge this PR so I can work on adding some options to this new menu to backup the current local cheats/patches to ZIP (see #11)

**Pull request merged**